config.php

```diff
@@ -193,9 +193,11 @@
  *
  * You can see the default implementations (and what gets passed in) in index.php.
  */
-//$min_factories['minify'] = ... a callable
-//$min_factories['controller'] = ... a callable
+//$min_factories['minify'] = ... a callable accepting a Minify\App object
+//$min_factories['controller'] = ... a callable accepting a Minify\App object
 
 
 // try to disable output_compression (may not have an effect)
-ini_set('zlib.output_compression', '0');
+if (!headers_sent()) {
+    ini_set('zlib.output_compression', '0');
+}
```

Comment:
i never liked the business logic in config file, why this has to stay in config?